### PR TITLE
feat: add ::backdrop using MDN data

### PR DESCRIPTION
**Pull request by yisibl**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+.DS_Store
```
Comment (Semigradsky) [inline .gitignore:1]: Why not in your own global .gitignore?

Comment (ai) [reply .gitignore:1]: I agree. These files should be in global git ignore.

Comment (yisibl) [reply .gitignore]: Done

**Comment by ai** [comment]
Sorry, I am on the conference right now and will need a few days for review

**Comment by yisibl** [comment]
@ai it's all right.

**Pull request merged**